modin/backends/pandas/query_compiler.py

```diff
@@ -1488,9 +1479,6 @@ def datetime_freq(df):
     dt_days = MapFunction.register(_dt_prop_map("days"))
     dt_microseconds = MapFunction.register(_dt_prop_map("microseconds"))
     dt_nanoseconds = MapFunction.register(_dt_prop_map("nanoseconds"))
-    dt_components = MapFunction.register(
```

Comment:
Is this being removed since we no longer support the `validate_columns` argument?